sci-biology/gffutils/gffutils-0.8.3.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="GFF and GTF file manipulation and interconversion"
+HOMEPAGE="https://pythonhosted.org/gffutils/"
+SRC_URI="https://github.com/daler/gffutils/archive/v${PV}.tar.gz"
```

Comment:
you need to rename the tarball to avoid name clashes

    SRC_URI="https://github.com/daler/gffutils/archive/v${PV}.tar.gz -> ${P}.tar.gz"